internal/pkg/manifest/app_manifest.go

```diff
@@ -35,12 +35,19 @@ type AppImage struct {
 	Build string `yaml:"build"` // Path to the Dockerfile.
 }
 
+// CreateAppManifestProps contains all properties for creating a new manifest.
+type CreateAppManifestProps struct {
+	AppName    string
+	Dockerfile string
+	Path       string
```

Comment:
The problem with `Path` is that it's a field that's only particular to the "load balanced web app" pattern. So I don't think it should belong to this common data structure. Maybe we are overcomplicating and don't need this method yet since we have a single app type. 

Let's just `NewLoadBalancedFargateManifest` from `app_init.go` and remove this method.